pkg/restore/restore.go

```diff
@@ -928,11 +932,83 @@ func (ctx *restoreContext) itemsAvailable(action framework.RestoreItemResolvedAc
 	return available, err
 }
 
-func (ctx *restoreContext) getResourceClient(groupResource schema.GroupResource, obj *unstructured.Unstructured, namespace string) (client.Dynamic, error) {
-	key := resourceClientKey{
+func (ctx *restoreContext) processPagerClientCalls(gr schema.GroupResource, label string, resourceClient client.Dynamic) (runtime.Object, error) {
```

Comment:
This func is copied almost verbatim from item_collector. If we end up using it, we'll want to refactor it into a shared util func somewhere.